prisma/migrations/20230524083610_init/migration.sql

```diff
@@ -0,0 +1,56 @@
+-- CreateTable
```

Comment:
We should create separate migrations for related tasks. Doing everything in a single migration is not a good practise. Let's say if we have to revert a single query then in this case we can't do that.